### ISSUE TITLE
Update `POWERLEVEL9K_BACKGROUND_JOBS` default colors

### ISSUE THREAD

**Issue by lucatrv** [bhilburn/powerlevel9k#868]
The default `POWERLEVEL9K_BACKGROUND_JOBS_BACKGROUND` and `POWERLEVEL9K_BACKGROUND_JOBS_FOREGROUND` colors are hardly visible. I would suggest to update them as follows, which stand out much better on both dark and light terminal themes:

```
POWERLEVEL9K_BACKGROUND_JOBS_FOREGROUND='black'
POWERLEVEL9K_BACKGROUND_JOBS_BACKGROUND='yellow'
```

If you agree I can create a PR.

**Comment by dritter** [comment]
Hi @lucatrv !

Sorry for the late response. That depends very much on the used color scheme. The `background_jobs` segment is cyan on black/white (depends on `$POWERLEVEL9K_COLOR_SCHEME`)..
Could you post a screenshot with a default color scheme so we can see the segment?

**Comment by lucatrv** [comment]
I attach the screenshots with Powerlevel9k default color scheme, on Gnome Terminal with both dark and light themes.

Default on Gnome Terminal dark:
![default_on_dark](https://user-images.githubusercontent.com/8380499/44815763-b6a39d00-abe0-11e8-8584-b671542f9d84.png)

Default on Gnome Terminal light:
![default_on_light](https://user-images.githubusercontent.com/8380499/44815773-bd321480-abe0-11e8-9e53-23524a6e8d6f.png)

Proposed on Gnome Terminal dark:
![proposed_on_dark](https://user-images.githubusercontent.com/8380499/44815787-c1f6c880-abe0-11e8-97c2-48832d98f1c2.png)

Proposed on Gnome Terminal light:
![proposed_on_light](https://user-images.githubusercontent.com/8380499/44815808-d0dd7b00-abe0-11e8-89f7-aa1ea92e7c82.png)

**Comment by bhilburn** [comment]
I actually think is a good idea. Sorry it took so long to get a response, @lucatrv.

Please feel free to submit a PR against the `next` branch. Note that the codebase on `next` is very different from what's currently on `master`, so you'll need to make the change on that code and file the PR `=)`

**Comment by lucatrv** [comment]
@bhilburn while working on this PR, I found out that to get the following colors:

```
POWERLEVEL9K_BACKGROUND_JOBS_FOREGROUND='black'
POWERLEVEL9K_BACKGROUND_JOBS_BACKGROUND='yellow'
```

I need to edit the file `segments/background_jobs.p9k` as follows

```
# Register segment
# Parameters:
#   segment_name  context  foreground  background  Generic  Flat/Awesome-Patched  Awesome-FontConfig  Awesome-Mapped-FontConfig  NerdFont
#                                                            ⚙                                                               
p9k::register_segment 'BACKGROUND_JOBS' '' "yellow" "black" $'\u2699'  $'\uE82F '  $'\uF013 '  '\u'$CODEPOINT_OF_AWESOME_COG' '  $'\uF013 '
```

while I would have expected:

```
# Register segment
# Parameters:
#   segment_name  context  foreground  background  Generic  Flat/Awesome-Patched  Awesome-FontConfig  Awesome-Mapped-FontConfig  NerdFont
#                                                            ⚙                                                               
p9k::register_segment 'BACKGROUND_JOBS' '' "black" "yellow" $'\u2699'  $'\uE82F '  $'\uF013 '  '\u'$CODEPOINT_OF_AWESOME_COG' '  $'\uF013 '
```

(`foreground` and `background` colors seem to be swapped).

Can you please advise?

**Comment by bhilburn** [comment]
Hm, really? @onaforeignshore - Can you comment, here?

@dritter - You haven't seen any color reversals have you?

**Comment by lucatrv** [comment]
Note: I'm on Debian Stable, zsh v5.3.1

**Comment by dritter** [comment]
Hmm. Changing the fore and background color works for me. The difference to `v0.6.6` is that you need to start a new shell to see your changes. It is not possible to change the colors on the fly any more.

**Comment by lucatrv** [comment]
This is really weird... I now installed zsh v5.6.2 from Debian Testing, and restarted Gnome Terminal, and yet in order to get the desired colors:

![image](https://user-images.githubusercontent.com/8380499/46768361-2f217180-cce8-11e8-9b23-b841b79bc462.png)

I need to edit `segments/background_jobs.p9k` as follows:

```
# Register segment
# Parameters:
#   segment_name  context  foreground  background  Generic  Flat/Awesome-Patched  Awesome-FontConfig  Awesome-Mapped-FontConfig  NerdFont
#                                                            ⚙                                                               
p9k::register_segment 'BACKGROUND_JOBS' '' "yellow" "black" $'\u2699'  $'\uE82F '  $'\uF013 '  '\u'$CODEPOINT_OF_AWESOME_COG' '  $'\uF013 '
```

(`foreground` and `background` colors are swapped).

Not sure what I'm doing wrong...

**Comment by dritter** [comment]
Indeed, that comment is wrong, if you look at the definition of `p9k::register_segment`, you'll see the following comment:
```
# @args
#   $1 string Segment name
#   $2 string State name or ""
#   $3 misc Default background color
#   $4 misc Default foreground color
#   $5 string Generic icon
#   $6 string Flat / Awesome Patched icon
#   $7 string Awesome FontConfig icon
#   $8 string Awesome Mapped FontConfig icon
#   $9 string NerdFont Complete / FontConfig icon
```
So the third parameter is the background color and fourth is foreground color..

**Comment by lucatrv** [comment]
I see. Should I open an issue about this, or submit a PR, or will you take care yourself?

**Comment by lucatrv** [comment]
Or I can add a commit to my PR

**Comment by dritter** [comment]
I'd prefer a separate PR :)
Good catch, btw.

**Comment by bhilburn** [comment]
Fixed by #1014! Thanks so much, @lucatrv! =)
